python-shell/src/gafferpy/gaffer_core.py

```diff
@@ -157,15 +157,8 @@ class ResultsWanted:
     RELATED_ONLY = 'RELATED_ONLY'
     BOTH = 'BOTH'
 
-class MergeType:
```

Comment:
The code above this, the results wanted enum needs removing.

Reply:
Ah yeah, thanks for spotting this